be/src/storage/fs/file_block_manager.cpp

```diff
@@ -352,19 +350,19 @@ Status FileBlockManager::create_block(const CreateBlockOptions& opts, std::uniqu
 Status FileBlockManager::open_block(const std::string& path, std::unique_ptr<ReadableBlock>* block) {
     VLOG(1) << "Opening block with path at " << path;
     std::shared_ptr<OpenedFileHandle<RandomAccessFile>> file_handle(new OpenedFileHandle<RandomAccessFile>());
-    bool found = _file_cache->lookup(path, file_handle.get());
-    if (!found) {
+    if (_file_cache != nullptr && !_file_cache->lookup(path, file_handle.get())) {
```

Comment:
Under what circumstance, the _file_cache will be a null pointer?

Reply:
_fs->type() != FileSystem::TYPE_POSIX